academy/3-my-own-chain/migration.md

```diff
@@ -1630,7 +1661,11 @@ $ docker exec -it checkers ./release/v1/checkersd start \
     --rpc.laddr "tcp://0.0.0.0:26657"
 ```
 
-Note that you need to force the node to listen on all IP addresses, not just `127.0.0.1` as it would do by default.
+<HighlightBox type="note">
```

Comment:
I think that it will not show as a highlight box because it is already in a `CodeGroupItem`.

Reply:
Corrected.